package.json

```diff
@@ -191,5 +191,9 @@
     "@elastic/apm-rum-core": "file:packages/rum-core",
     "@elastic/apm-rum-react": "file:packages/rum-react",
     "@elastic/apm-rum-vue": "file:packages/rum-vue"
+  },
+  "engines": {
+    "node": ">=12 <=16",
```

Comment:
Can we also change the CI to use node 12 images? 

https://github.com/elastic/apm-agent-rum-js/blob/07bc85f7ea9fb036063faea9d03a4535ee18df7f/Jenkinsfile#L89

Reply:
good point!